sdk/python/sawtooth_sdk/client/stream.py

```diff
@@ -46,17 +50,24 @@ class _SendReceiveThread(Thread):
     Internal thread to Stream class that runs the asyncio event loop.
     """
 
-    def __init__(self, url, futures):
+    def __init__(self, url, futures, ready_event):
+        """constructor for background thread
+
+        :param url (str): the address to connect to the validator on
+        :param futures (FutureCollection):
+        :param ready_event (threading.Event):
```

Comment:
Should fill out this API doc.  In particular, some description of ready_event works would be useful.